data/data-pipeline/data_pipeline/ipython/scoring_comparison.ipynb

```diff
@@ -1287,7 +1332,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.6"
+   "version": "3.6.2"
```

Comment:
I am using `pyenv` here and the reference to this version number reflects compatibility for a system level version. I needed to use `3.6.2` for certain spatial statistic packages.